### PR TITLE
Added Discord url @ faucet channel mention

### PR DESCRIPTION
**Pull request by cuddleofdeath**

### CODE REVIEW

File: docs/developers/wallet.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ celestia-appd keys list
 ## Fund a Wallet
 
 For the public celestia address, you can fund the
-previously created wallet via Discord by sending
+previously created wallet via [Discord](https://discord.com/invite/YsnTPcSfWQ) by sending
```
Comment (qubelabsio) [inline docs/developers/wallet.md:30]: Can you please use our global link instead ->  https://discord.gg/celestiacommunity

**Review by qubelabsio — CHANGES_REQUESTED** [review]

File: docs/developers/wallet.md (hunk 1 above)
Comment (cuddleofdeath) [reply docs/developers/wallet.md:30]: For sure! I'll get that fixed momentarily. I grabbed the original link from the Discord button at the bottom of Celestia's website.

**Review by adlerjohn — APPROVED** [review]

File: docs/developers/wallet.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ celestia-appd keys list
 ## Fund a Wallet
 
 For the public celestia address, you can fund the
-previously created wallet via Discord by sending
+previously created wallet via [Discord](https://discord.gg/celestiacommunity) by sending
```
Comment (YazzyYaz) [inline docs/developers/wallet.md:30]: Can you fix the lint errors?

You can do that by moving "by sending" to the next line so to not exceed 80 chars per line. 

Also make sure there's no blank space at the end of each line you added.

Comment (cuddleofdeath) [reply docs/developers/wallet.md]: Thank you for the feedback, I just realized after reading this comment that I didn't have the markdownlint extension installed on VSCode! Just installed it, hope all is good now!

Here is the new PR following the steps outlined in the comment!:
https://github.com/celestiaorg/docs/pull/195/commits/707aa91e094284d0aeae838c99f8402963a47961

**Review by adlerjohn — APPROVED** [review]

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**